src/initialize-app.tsx

```diff
@@ -0,0 +1,108 @@
+import "ts-polyfill";
+
+import React, { useEffect } from "react";
+import { Provider } from "mobx-react";
+import { setLivelinessChecking } from "mobx-state-tree";
+import Modal from "react-modal";
+import { ModalProvider } from "@concord-consortium/react-modal-hook";
+import { QueryClient, QueryClientProvider } from "react-query";
+
+import { AppMode } from "./models/stores/store-types";
+import { Logger } from "./lib/logger";
+import { appConfigSnapshot, appIcons, createStores } from "./app-config";
+import { AppConfigContext } from "./app-config-context";
+import { AppConfigModel } from "./models/stores/app-config-model";
+import { IStores, setUnitAndProblem } from "./models/stores/stores";
+import { UserModel } from "./models/stores/user";
+import { urlParams } from "./utilities/url-params";
+import { DEBUG_STORES } from "./lib/debug";
+import { gImageMap } from "./models/image-map";
+import PackageJson from "./../package.json";
+
+import "./index.scss";
+
+// set to true to enable MST liveliness checking
+const kEnableLivelinessChecking = false;
+
+export const appConfig = AppConfigModel.create(appConfigSnapshot);
+
+// TODO: we might as well just return
+// stores instead of an object container stores
+export interface IAppProperties {
+  stores: IStores;
+}
```

Comment:
Seems like this wouldn't be too difficult to change. Should we just take care of it now?